src/utils/colonyActions.ts

```diff
@@ -487,6 +488,28 @@ export const getDetailsForAction = (
 //   }
 // };
 
+export const normalizeRolesForAction = (
+  roles: ColonyActionRoles,
+): ActionUserRoles[] => {
+  /*
+   * Done manually since this list is static
+   */
+  const extractedRoles = [
+    { id: 0, setTo: roles.role_0 },
+    { id: 1, setTo: roles.role_1 },
+    { id: 2, setTo: roles.role_2 },
+    { id: 3, setTo: roles.role_3 },
+    { id: 5, setTo: roles.role_5 },
+    { id: 6, setTo: roles.role_6 },
+  ];
+  return extractedRoles.filter(
+    ({ setTo }) => setTo !== null,
```

Comment:
Just making sure this is right, you're not excluding the case in which `setTo` is undefined (and looking at the type it could be)